dependencies.gradle

```diff
@@ -50,7 +50,7 @@ ext.libs = [
                 'coroutinesTest'          : "org.jetbrains.kotlinx:kotlinx-coroutines-test:$kotlinCoroutines"
         ],
         androidx    : [
-                'activity'                : "androidx.activity:activity:1.5.0",
+                'activity'                : "androidx.activity:activity:1.5.1",
```

Comment:
Can we reuse `$fragment` here? (and rename the var?). Related to #6657

Reply:
I am not sure we should. Since it seems they don't follow the same release dates and versionning. Comparing the versionning history in https://developer.android.com/jetpack/androidx/releases/fragment and in https://developer.android.com/jetpack/androidx/releases/activity, we can see version 1.4.X is different for example.